### PR TITLE
fix(cache): remove cache rules from fetch()

### PR DESCRIPTION
**Pull request by avivkeller**
## Description

> Interesting, I think I know what might be happening. Forcing a refresh on local development did work, but that's because it invalidates the browser cache...

Posted by @ovflowd on Slack.

## Validation

**Hopefully,** the preview for this should show all blog posts, and once merged, the main page should do the same.

Fixes https://openjs-foundation.slack.com/archives/CVAMEJ4UV/p1732148207813379

### CODE REVIEW

File: apps/site/next-data/blogData.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ const getBlogData = (cat: string, page?: number): Promise<BlogPostsRSC> => {
   // When we're on RSC with Server capabilities we prefer using Next.js Data Fetching
   // as this will load cached data from the server instead of generating data on the fly
   // this is extremely useful for ISR and SSG as it will not generate this data on every request
-  return fetch(fetchURL, { cache: 'force-cache' })
+  return fetch(fetchURL)
```
Comment (ovflowd) [inline apps/site/next-data/blogData.ts:30]: Should we remove the other fetch() cache statements, I wonder?

Comment (avivkeller) [reply apps/site/next-data/blogData.ts:30]: Yes. Linting was taking forever, so I figured I'd do one to make sure it worked. I'm linting the others (I might just --no-verify at this point)

Comment (ovflowd) [reply apps/site/next-data/blogData.ts:30]: Since the pages themselves are cached we really don't need'em

Comment (avivkeller) [reply apps/site/next-data/blogData.ts:30]: I feel like we should keep it uniform.

Comment (bmuenzenmeyer) [reply apps/site/next-data/blogData.ts:30]: the comment should likely be updated too once we stabilize

Comment (avivkeller) [reply apps/site/next-data/blogData.ts:30]: I've updated it. Feel free to push to this branch if any of you want to change anything. You know more about caching in this codebase than I.

**Review by bmuenzenmeyer — APPROVED** [review]
the content looks right to me on the preview - do we have a sense for whether or not we can truly test this prior to release? the deployments themselves dont have a cache or a new release coming in.

also, I want to keep an eye on that lighthouse - hopefully it's an outlier `87`

**Comment by avivkeller** [comment]
> also, I want to keep an eye on that lighthouse - hopefully it's an outlier `87`

I just ran a lighthouse test (https://pagespeed.web.dev/analysis/https-nodejs-npvr1vjpb-openjs-vercel-app-en/yc38j12yqb). I got a 96 on desktop, 73 on mobile. Although the values seemed to fluctuate

**Comment by bmuenzenmeyer** [comment]
next discussion - is this critical enough a bug to fast-track? i would default towards no

**Comment by avivkeller** [comment]
I don't think so.

**Comment by ovflowd** [comment]
> I don't think so.

I do think so! Blog posts are missing on production /blog/ listing which is bad.

**Review by ovflowd — APPROVED** [review]

**Comment by ovflowd** [comment]
I'm fast tracking!

**Pull request merged**